pkg/osvscanner/osvscanner.go

```diff
@@ -827,25 +820,11 @@ func DoScan(actions ScannerActions, r reporter.Reporter) (models.VulnerabilityRe
 		}
 		onlyUncalledVuln = onlyUncalledVuln && vuln
 
-		switch {
-		case !vuln && !onlyUncalledVuln && !licenseViolation:
+		if !vuln && !onlyUncalledVuln && !licenseViolation {
```

Comment:
Is this logic correct? i.e. if we only have uncalled vulns, 

```
vulns = true
onlyUncalledVuln = true
```

then this will fail this check? 

This this be:

```
if (!vuln || onlyUncalledVuln) && !licenseViolation {
}
```

?

Reply:
Yeah you're right, I'll make that change